### ISSUE TITLE
【PaddlePaddle Hackathon】62、在Paddle2ONNX 新增11个 Paddle 2.0 API 支持

### ISSUE THREAD

**Issue by TCChenlong** [PaddlePaddle/Paddle2ONNX#342]
（此 ISSUE 为 PaddlePaddle Hackathon 活动的任务 ISSUE，更多详见[PaddlePaddle Hackathon](https://www.paddlepaddle.org.cn/PaddlePaddleHackathon)）

【任务说明】

- 任务标题：Paddle2ONNX 新增11个 Paddle 2.0 API 支持

- 技术标签：深度学习框架，模型转换，ONNX

- 任务难度：中等

- 详细描述：Paddle2ONNX为模型转换工具，负责将paddle的inference模型转换为ONNX格式，方便开发者将Paddle模型与其他基于ONNX的推理框架配合使用。Paddle2ONNX的底层将Paddle op逐一转换为ONNX对应op，最终生成ONNX格式模型。请帮助 Paddle2ONNX增加11个 OP支持。在转换过程中，支持多个ONNX版本协议，以提升飞桨模型适配硬件的能力.

  11个 API 具体为：

  paddle.nn.functional.interpolate(x, size=None, scale_factor=None, mode='nearest', align_corners=False, align_mode=0, data_format='NCHW', name=None) 

  paddle.nn.functional.softshrink(x, threshold=0.5, name=None) 

  paddle.tan(x, name=None) 

  paddle.nn.Tanhshrink(name=None) 

  paddle.nn.functional.thresholded_relu(x, threshold=1.0, name=None) 

  paddle.unique(x, return_index=False, return_inverse=False, return_counts=False, axis=None, dtype='int64', name=None) 

  paddle.where(condition, x, y, name=None) 

  paddle.scatter(x, index, updates, overwrite=True, name=None) 

  paddle.scatter_nd_add(x, index, updates, name=None) 

  paddle.meshgrid(*args, **kargs) 

  paddle.masked_select(x, mask, name=None) 

【提交内容】

- 任务 PR到 [Paddle2ONNX](https://github.com/PaddlePaddle/Paddle2ONNX)

- 相关技术文档

- 任务单测文件

【技术要求】

- 了解飞桨框架使用

- 了解ONNX
